### PR TITLE
fix: Rename callback to more descriptive name. Also callback should be…

### PR DESCRIPTION
**Pull request by nilscb**
… triggered when deck.gl receives an empty layer list.

### CODE REVIEW

File: typescript/packages/subsurface-viewer/src/components/Map.tsx [hunk 1]
```diff
@@ -917,13 +918,17 @@ const Map: React.FC<MapProps> = ({
...
         if (deckGLLayers) {
-            const state = deckGLLayers.every((layer) => {
+            const loadedState = deckGLLayers.every((layer) => {
                 return (layer as Layer).isLoaded;
             });
-            setIsLoaded(state);
 
+            const emptyLayers = // There will always be a dummy layer. Deck.gl does not like empty array of layers.
+                deckGLLayers.length == 1 &&
+                (deckGLLayers[0] as LineLayer).id === "dummy";
```
Comment (t0oF-azpn) [inline typescript/packages/subsurface-viewer/src/components/Map.tsx:927]: Rename to "webviz_internal_dummy_layer"

File: typescript/packages/subsurface-viewer/src/components/Map.tsx [hunk 2]
```diff
@@ -396,7 +396,7 @@ export interface MapProps {
     /**
      * Will be called after all layers have finished loading data.
```
Comment (t0oF-azpn) [inline typescript/packages/subsurface-viewer/src/components/Map.tsx:397]: loading data => rendering

**Comment by hkfb** [comment]
LGTM, but I don't see any "new wellsLayer property" as mentioned in the description.

**Review by t0oF-azpn — APPROVED** [review]

**Review by hkfb — APPROVED** [review]

**Pull request merged**